qlkube/deploy/qlkube/Chart.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: qlkube
+description: A Helm chart for Kubernetes
```

```suggestion
description: The component exposing the Kubernetes resources through graphql
```

Or something like that